### PR TITLE
adjustments to native ios part

### PR DESCRIPTION
**Pull request by BondarenkoStas**

### CODE REVIEW

File: content/yaml/building-a-native-ios-app.md [hunk 1]
```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
                        CODE_SIGNING_REQUIRED=NO \
                        CODE_SIGNING_ALLOWED=NO
 
+If you don't have a workspace, use `-project "MyXcodeProject.xcodeproj"` instead of the `-workspace "MyXcodeWorkspace.xcworkspace"` option.
```
Comment (priitlatt) [inline content/yaml/building-a-native-ios-app.md:25]: I cannot comment on non-changed lines, but the example script starts with `cd` statement to build dir, which is completely unnecessary as all commands are executed by default with `$FCI_BUILD_DIR` being the working directory.

File: content/yaml/building-a-native-ios-app.md [hunk 2]
```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
...
+
+Your artifact will be generated at the default Xcode path (unless you explicitly specify another path using the `CONFIGURATION_BUILD_DIR` option) by adding the following pattern in the `artifacts` section of `codemagic.yaml`:
```
Comment (priitlatt) [inline content/yaml/building-a-native-ios-app.md:27]: > Your artifact will be generated at the default Xcode path /.../ by adding the following pattern in the artifacts section of ...

This doesn't make sense. It implies that specifying the pattern in artifact paths somehow makes Xcode to save the artifact there.

File: content/yaml/building-a-native-ios-app.md [hunk 3]
```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
...
+    $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM
+
 ## Building a native iOS app archive (.ipa)
 
 For building an archived iOS app (.ipa), you need to run the following command in the scripts section:
 
     - xcode-project build-ipa --project "$FCI_BUILD_DIR/MyXcodeProject.xcodeproj" --scheme "MyScheme"
 
+You can change the default artifact location `build/ios/ipa` by specifying [`--ipa-directory`](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/build-ipa.md#--ipa-directoryipa_directory) option. Xcode build log can be found with `/tmp/xcodebuild_logs/*.log` pattern, and dSYM is still available with default Xcode path:
+
```
Comment (priitlatt) [inline content/yaml/building-a-native-ios-app.md:42]: For clarity I'd prefer to have it as full block that is expected in Yaml:

```yaml
- artifacts:
    - build/ios/ipa/*.ipa
    - /tmp/xcodebuild_logs/*.log
    ...
```

File: content/yaml/building-a-native-ios-app.md [hunk 4]
```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
...
+If you have Xcode Debugging Symbols enabled, dSYM file will be generated in the same directory as the app and can be accessed with pattern:
+
+    $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM
+
 ## Building a native iOS app archive (.ipa)
 
 For building an archived iOS app (.ipa), you need to run the following command in the scripts section:
 
     - xcode-project build-ipa --project "$FCI_BUILD_DIR/MyXcodeProject.xcodeproj" --scheme "MyScheme"
```
Comment (priitlatt) [inline content/yaml/building-a-native-ios-app.md:39]: The prefix `$FCI_BUILD_DIR` in path `$FCI_BUILD_DIR/MyXcodeProject.xcodeproj` is redundant for the same reason as stated above.

Comment (priitlatt) [inline content/yaml/building-a-native-ios-app.md:39]: This example only covers the possibility to use Xcode projecet, but actually one can build ipas from workspaces too. See [--project](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/build-ipa.md#--projectxcode_project_path) and [--workspace](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/build-ipa.md#--workspacexcode_workspace_path) arguments.

File: content/yaml/building-a-native-ios-app.md (hunk 1 above)
Comment (BondarenkoStas) [reply content/yaml/building-a-native-ios-app.md:23]: thank you! Fixed in 9f13727, as well as some other places where it's not redundant. I just remember I wrote something like `<you_project_directory>` but it was changed:)

File: content/yaml/building-a-native-ios-app.md (hunk 2 above)
Comment (BondarenkoStas) [reply content/yaml/building-a-native-ios-app.md]: fixed here 79cf5b8

File: content/yaml/building-a-native-ios-app.md (hunk 3 above)
Comment (BondarenkoStas) [reply content/yaml/building-a-native-ios-app.md:46]: fixed here 79cf5b8

File: content/yaml/building-a-native-ios-app.md (hunk 4 above)
Comment (BondarenkoStas) [reply content/yaml/building-a-native-ios-app.md]: fixed in 9f13727

Comment (BondarenkoStas) [reply content/yaml/building-a-native-ios-app.md]: added here 79cf5b8

**Pull request merged**